github/repos_test.go

```diff
@@ -1066,13 +1066,13 @@ func TestRepositoriesService_GetSignaturesProtectedBranch(t *testing.T) {
 	})
 
 	signature, _, err := client.Repositories.GetSignaturesProtectedBranch(context.Background(), "o", "r", "b")
+
```

Comment:
Please delete this blank line, as we attempt to keep the error checking code tightly bound to the section of code that created it.